### ISSUE TITLE
RC-switch library

### ISSUE THREAD

**Issue by danricho** [dgomes/homeGW#12]
is there any way to allow the [RC-SWITCH](https://github.com/sui77/rc-switch) library to work alongside HomeGW using the same RF receiver?

I have some remote controls which are received by RC-switch but are not with the generic raw plugin.

**Comment by dgomes** [comment]
Unfortunately no, both rely in the interrupt functionality which cannot be shared amongst libraries.

It would take a full rewrite of both libraries to make that possible.

**Comment by danricho** [comment]
I got it working by wrapping both libraries’ interrupt ISR, with a new one which subsequently calls both.

I also tweaked both libraries to make the built in ISRs public.

After running the setup method I detached the interrupt and remapped it to my new ISR.

I think a nice solution might be to accept an argument to the setup function which stops the interrupt attachment.

RC-Switch doesn’t attach until another function is called (I can’t remember the name and I’m away from my computer).

Thanks for the great library!

Dan
